open-cloud-dashboards.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+gp preview $(aws-vault login default) --external
```

Comment:
This is actually Gitpod specific, we'll want ways to detect the environment we're running in, and offer different options depending on the environment. Could leave as this for now, but then fix when we add devcontainer, and/or codespaces configurations.